adafruit_ags02ma.py

```diff
@@ -80,30 +118,40 @@ def __init__(self, i2c_bus, address=AGS02MA_I2CADDR_DEFAULT):
         except RuntimeError as exc:  # a CRC error or something!
             raise RuntimeError("Failed to find AGS02MA - check your wiring!") from exc
 
-    def firmware_version(self):
+    def firmware_version(self) -> int:
         """Return 24-bit value which contains the firmware version"""
         return self._read_reg(_AGS02MA_VERSION_REG, 30)
 
     @property
-    def gas_resistance(self):
+    def gas_resistance(self) -> float:
         """The resistance of the MEMS gas sensor"""
         return self._read_reg(_AGS02MA_GASRES_REG, 1500) * 100  # in 0.1Kohm
```

Comment:
Does this get converted to a float somehow? 

`self._read_reg()` is typed below as `int` and multiplying an int by 100 should result in another int I think.

Reply:
Corrected pls see https://github.com/adafruit/Adafruit_CircuitPython_AGS02MA/pull/3/commits/7d2eb7a3242cd3fcb3af70b409ba43b5e1fec6b6